apps/server/src/modules/tldraw/service/tldraw.service.spec.ts

```diff
@@ -2,20 +2,26 @@ import { Test, TestingModule } from '@nestjs/testing';
 import { EntityManager } from '@mikro-orm/mongodb';
 import { MongoMemoryDatabaseModule } from '@infra/database';
 import { cleanupCollections } from '@shared/testing';
+import { ConfigModule } from '@nestjs/config';
+import { createConfigModuleOptions } from '@src/config';
 import { TldrawDrawing } from '../entities';
 import { tldrawEntityFactory } from '../factory';
 import { TldrawRepo } from '../repo/tldraw.repo';
 import { TldrawService } from './tldraw.service';
+import { config } from '../config';
 
-describe(TldrawService.name, () => {
+describe('TldrawService', () => {
 	let module: TestingModule;
 	let service: TldrawService;
 	let repo: TldrawRepo;
 	let em: EntityManager;
 
 	beforeAll(async () => {
 		module = await Test.createTestingModule({
-			imports: [MongoMemoryDatabaseModule.forRoot({ entities: [TldrawDrawing] })],
+			imports: [
+				MongoMemoryDatabaseModule.forRoot({ entities: [TldrawDrawing] }),
+				ConfigModule.forRoot(createConfigModuleOptions(config)),
```

Comment:
Check my comment on other config setup in test

Reply:
here we cannot use deepmocked as we have to check if entity was removed from database